llvm-spirv/lib/SPIRV/OCLUtil.cpp

```diff
@@ -531,6 +531,23 @@ class OCLBuiltinFuncMangleInfo : public SPIRV::BuiltinFuncMangleInfo {
       addVoidPtrArg(1);
       addUnsignedArg(2);
       addUnsignedArg(3);
+      // TODO: REMOVE BEFORE COMMIT!!!
+      // Here and after we are translating calls of blocking pipe SPIR-V
+      // instructions to nonexistent OpenCL built-ins calls, so LLVM IR before
+      // and after translation changes (actually same is happening for
+      // non-blocking pipes, which is considered as a bug that must be fixed).
+      // Instead of it we can not create special rules for ReadPipeBlockingINTEL
```

Comment:
not -> now, if this comment will remain

Reply:
It's "can not" (with a pause between the words). I think "and WritePipeBlockingINTEL **AND** emit SPIR-V looking like LLVM IR" should be changed to "and WritePipeBlockingINTEL **BUT** emit SPIR-V looking like LLVM IR".

Reply:
Ah, yes I understand it a little better with the alternative phrasing! Thank you.

Reply:
Hope it's better now.